### PR TITLE
Fixes for tasklist

### PR DESCRIPTION
**Pull request by AtosSK**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,7 @@ appsettings.Test.json
 Nu[Gg]et.[Cc]onfig
 wwwroot/
 /src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/wwwroot/js/index.js.map
+/src/NuGet.Config
```
Comment (NikPeters-Atos) [inline .gitignore]: Why are you making changes to this file?

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs [hunk 2]
```diff
@@ -172,6 +172,23 @@ public async Task<IActionResult> OperatorType(OperatorTypeViewModel vm)
             return NotFound();
         }
 
+        [HttpGet("Site/{siteId}/Material/{materialId}/TaskListSite", Name = "TaskListSite")]
+        public async Task<IActionResult> TaskListSite(
+            Guid? id,
+            Guid? materialId)
+        {
+            if (id != null &&
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: Is there a back button that needs populating?

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs [hunk 3]
```diff
@@ -172,6 +172,23 @@ public async Task<IActionResult> OperatorType(OperatorTypeViewModel vm)
             return NotFound();
         }
 
+        [HttpGet("Site/{siteId}/Material/{materialId}/TaskListSite", Name = "TaskListSite")]
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: Why does site id exist, yet not being used? Is this required for the next step? Is site id required for the further journey? If so, is this required for both exporter and reprorcessor? Or just reprocessor? And if it's just reprocessor, then site is a one to one relationship with an accreditation, and therefore no site id is required

File: src/NuGet.Config [hunk 4]
```diff
@@ -4,4 +4,4 @@
         <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
         <add key="epr-packaging-common" value="https://pkgs.dev.azure.com/defragovuk/RWD-CPR-EPR4P-ADO/_packaging/defra-epr-common/nuget/v3/index.json" />
     </packageSources>
-</configuration>
```
Comment (NikPeters-Atos) [inline src/NuGet.Config:7]: Why are you making changes to the nuget.config file?

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs (hunk 2 above)
Comment (AtosSK) [reply src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: Done.

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs (hunk 3 above)
Comment (AtosSK) [reply src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: Done.

File: .gitignore (hunk 1 above)
Comment (AtosSK) [reply .gitignore]: It seems to be picking up this file.

**Review by NikPeters-Atos — APPROVED** [review]

**Pull request merged**